src/goto-checker/goto_symex_property_decider.cpp

```diff
@@ -11,6 +11,11 @@ Author: Daniel Kroening, Peter Schrammel
 
 #include "goto_symex_property_decider.h"
 
+#include <util/threeval.h>
+
+#include <solvers/prop/literal_expr.h>
+#include <solvers/prop/prop.h>
```

Comment:
With the idea implemented in smt2_dec.h I'd say `util/threeval.h` should thus be the last include?

Reply:
done